src/components/UserLogin/userlogin.css

```diff
@@ -0,0 +1,24 @@
+body {
+    background-color: '#fff'
+}
+
+.paper {
+    display: flex;
+    margin-top: 64px;
+    align-items: center;
+    flex-direction: column;
+}
+
+.avatar {
+    margin: 8px;
+    background-color: #f50057;
+}
+
+.form {
+    width: 100%;
+    margin-top: 8px;
+}
```

Comment:
Make the generic  selector nested 
Like .sign-in .form{}

Reply:
ok